RocketChat/Rocket.Chat.ReactNative
# [NEW] Unify members section

## Proposed changes

Unifies the session of adding or inviting users to the members screen.

## Issue(s)	


## How to test or reproduce

- Try to add or remove some user through the new menu.

Check if the settings are working correctly:
- Possibility to add or not a user.
- Possibility of inviting or not a user.
## Screenshots

## Types of changes




## Checklist



## Further comments
